crates/cli-support/src/js/mod.rs

```diff
@@ -3141,6 +3141,11 @@ impl<'a> Context<'a> {
                 format!("typeof({}) === 'function'", args[0])
             }
 
+            Intrinsic::IsArray => {
+                assert_eq!(args.len(), 1);
+                format!("{} instanceof Array", args[0])
```

Comment:
This should use [`Array.isArray`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) instead, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#instanceof_vs_isarray.